tox.ini

```diff
@@ -0,0 +1,30 @@
+; For all tests that run in Github Actions, see
+; .github/workflows/cpu-ci.yml for the workflow definition.
+
+[tox]
+envlist = python3.8,test-gpu
+
+[testenv]
+commands =
+    pip install --upgrade pip
+    pip install -e .[all]
+
+[testenv:test-gpu]
+passenv=
+    CUDA_VISIBLE_DEVICES
```

Comment:
Do we need this environment at all? It looks like it's set to `CUDA_VISIBLE_DEVICES=0,1` in Jenkins, but Models doesn't have mult-gpu support yet, and horovod, which we plan to use as the backed for distirubted training in Models, discourages the use of this environment variable. I'm wondering if existing tests still work if we remove it from Jenkins.